joblib/parallel.py

```diff
@@ -144,13 +144,13 @@ class parallel_backend(object):
 
     It is also possible to use the distributed 'ray' backend for distributing
     the workload to a cluster of nodes. To use the 'ray' joblib backend add
-    the following lines:
+    the following lines::
 
-    >> from ray.util.joblib import register_ray
-    >> register_ray()
-    >> with parallel_backend("ray"):
-    ..     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))
-    [-1, -2, -3, -4, -5]
+     >>> from ray.util.joblib import register_ray  # doctest: +SKIP
+     >>> register_ray()  # doctest: +SKIP
+     >>> with parallel_backend("ray"):  # doctest: +SKIP
+     ...     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))  # doctest: +SKIP
```

Comment:
To appease the linter, maybe deactivate warning on the long line:
```suggestion
     ...     print(Parallel()(delayed(neg)(i + 1) for i in range(5)))  # doctest: +SKIP  # noqa
```

Reply:
Thanks!!

Reply:
mmh, now, doctests are failing... https://github.com/PyCQA/pydocstyle/issues/234#issuecomment-294282208 suggest to put the `# noqa` one the line of the function definition.